exchanges/binance/binance_wrapper.go

```diff
@@ -956,10 +956,15 @@ func (b *Binance) SubmitOrder(ctx context.Context, s *order.Submit) (order.Submi
 			return submitOrderResponse, fmt.Errorf("invalid side")
 		}
 
-		var oType string
+		var (
+			oType       string
+			timeInForce string
+		)
+
 		switch s.Type {
 		case order.Limit:
 			oType = cfuturesLimit
+			timeInForce = "GTC"
```

Comment:
Can use our built-in time in force types:

```suggestion
			timeInForce = BinanceRequestParamsTimeGTC
```

Then this looks good to go! 

![image](https://user-images.githubusercontent.com/4685270/149427650-16ed235b-c2cc-4a62-a386-d14e90f208e8.png)

Reply:
Only thing to add is that in order to support this, you'll have to use `string(x)` at some point, which I'd suggest be inside `FuturesNewOrder` to support/use of  `RequestParamsTimeForceType` type
eg
```
if x.TimeInForce != "" {
		params.Set("timeInForce", string(x.TimeInForce))
	}
```
Like in `newOrder`

Reply:
sounds good, also ended up changing `FuturesNewOrderRequest` `TimeInForce` field to the `RequestParamsTimeForceType` type